BlueDB/src/main/java/org/bluedb/api/BlueDb.java

```diff
@@ -9,6 +9,8 @@
 	
 	public <T extends Serializable> BlueCollection<T> initializeCollection(String name, Class<? extends BlueKey> keyType, Class<T> valueType, @SuppressWarnings("unchecked") Class<? extends Serializable>... additionalClassesToRegister) throws BlueDbException;

```

Comment:
Yeah, I think we should deprecate it soon.

I prefer collectionBuilder to buildCollection because buildCollection seems to imply action taken when action isn't taken until build() is called.  I'd be open for discussion or other ideas though.